crates/astria-eyre/src/lib.rs

```diff
@@ -83,3 +85,44 @@ fn write_value(err: &dyn Error, f: &mut core::fmt::Formatter<'_>) -> core::fmt::
     f.write_fmt(format_args!("\"{err}\""))?;
     Ok(())
 }
+
+#[cfg(feature = "anyhow_support")]
```

Comment:
Put all of this under a module so you don't need to repeat `cfg(feature = ...)` all the time:

#[cfg(feature = "anyhow")]
pub use anyhow_conversion::{anyhow_to_eyre, eyre_to_anyhow};

```rust
#[cfg(feature = "anyhow")]
mod anyhow_conversion {
   pub fn anyhow_to_eyre() {}

   pub fn eyre_to_anyhow() {}
   
   #[cfg(test)]
    mod tests {
        // the tests
    }
}
```